.gitlab/custom-jobs-and-variables.yml

```diff
@@ -60,7 +60,7 @@ variables:
 # Project specific variants for lassen
   PROJECT_LASSEN_VARIANTS: "~shared +openmp +tests cuda_arch=70"
 # Project specific deps for lassen
-  PROJECT_LASSEN_DEPS: "^blt@git.a7f0a6ecc4fdfa1724399b1454c3909b9ee02e81=develop"
+  PROJECT_LASSEN_DEPS: "^blt@develop "
```

Comment:
@rhornung67 I'm guessing what you want is blt@6.1.0. I could update BLT in radiuss spack configs.

Reply:
I thought we wanted to be testing against BLT develop always.

Reply:
@adrienbernede I've started a PR to update our spack packages here as we are preparing releases: https://github.com/LLNL/radiuss-spack-configs/pull/90